pkgs/by-name/lo/lovr-playspace/package.nix

```diff
@@ -0,0 +1,59 @@
+# SPDX-FileCopyrightText: 2025 Red_M <nixpkgs-xr@red-m.net>
+#
+# SPDX-License-Identifier: MIT
+
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  symlinkJoin,
+  writeTextFile,
+  bash,
+  lovr,
+
+  xrSources,
+}:
+
+symlinkJoin rec {
+  inherit (xrSources.lovr-playspace)
+    pname
+    version
+    ;
+  paths = [
+    lovr
+    (stdenv.mkDerivation {
+      inherit (xrSources.lovr-playspace)
+        src
+        version
+        ;
+      pname = "${pname}-unwrapped";
+
+      dontUseCmakeConfigure = true;
+
+      installPhase = ''
+        runHook preInstall
+        mkdir -p $out/lovr-playspace $out/lovr-playspace/json
+        cp -r ./*.lua $out/lovr-playspace/
+        cp -r ./json/*.lua $out/lovr-playspace/json/
+        runHook postInstall
+      '';
+    })
+    (writeTextFile {
+      name = "lovr-playspace-script";
+      executable = true;
+      destination = "/bin/lovr-playspace";
+      text = ''
+        #!${bash}/bin/bash
+        ${lovr}/bin/lovr ${builtins.elemAt paths 1}/lovr-playspace
+      '';
+    })
+  ];
+  meta = with lib; {
```

Comment:
nit: I personally like to avoid `with;` anywhere where there could be confusion. I'd remove it here, but keep it for maintainers